DataFormats/TrackReco/interface/TrackExtraBase.h

```diff
@@ -73,12 +78,15 @@ class TrackExtraBase
 
     }
 
+    TrajParams const & trajParams() const  {return m_trajParams;}

```

Comment:
Maybe add a comment saying these values will not be set if the TrackExtra is read back from a file?